src/config/index.ts

```diff
@@ -24,17 +23,8 @@ export interface Config {
   };
 }
 
-let environment: Environment = 'local';
-
-if (!window) {
-  environment = 'mock';
-} else if (DAPP_HOSTS.testnet.includes(window.location.host)) {
-  environment = 'testnet';
-} else if (DAPP_HOSTS.preview.includes(window.location.host)) {
-  environment = 'preview';
-} else if (DAPP_HOSTS.mainnet === window.location.host) {
-  environment = 'mainnet';
-}
+const environment: Environment =
+  (process.env.REACT_APP_ENVIRONMENT as Environment | undefined) || 'testnet';
```

Comment:
Just for safety, I'd default to `mainnet`:
```suggestion
  (process.env.REACT_APP_ENVIRONMENT as Environment | undefined) || 'mainnet';
```